docs/IJSValueReader-api-windows.md

```diff
@@ -7,56 +7,56 @@ Kind: `interface`
+`IJSIValueReader` is used to read data from JavaScript in custom native modules.  It acts as a stream and supports all the types in [`JSValueType`](JSValueType).
 
 ## Properties
 ### ValueType
 `readonly`  [`JSValueType`](JSValueType) `ValueType`
 
-Gets the type of the current value.
+Returns the type of the current value.
 
 
 
 ## Methods
 ### GetBoolean
 bool **`GetBoolean`**()
 
-Gets the current `Boolean` value.
+Gets the current boolean value.
 
 
 
 ### GetDouble
 double **`GetDouble`**()
 
-Gets the current `Number` value as a `Double`.
+Gets the current number value as a double.
 
 
 
 ### GetInt64
 int64_t **`GetInt64`**()
 
-Gets the current `Number` value as an `Int64`.
+Gets the current number value as an int.
 
 
 
 ### GetNextArrayItem
 bool **`GetNextArrayItem`**()
 
-Gets the next array item and makes its value to be the current value.<br/><br/>It returns **`true`** if the next array item is acquired successfully. Otherwise, it returns **`false`**, and it means that reading of the JSON-like array is completed.<br/><br/>**Note**<br/>- Use [`ValueType`](#valuetype) to get the type of the array item and other GetXXX methods to read it.<br/>- Use [`GetNextObjectProperty`](#getnextobjectproperty) method to start reading property value of type`JSValueType::Object`.<br/>- Use [`GetNextArrayItem`](#getnextarrayitem) method to start reading property value of type`JSValueType::Array`
+Moves the reader to the next array item.  Returns if there is another item in the array.

```

Comment:
As above, it looks like there is content lost - is it surfaced elsewhere or is it unnecessary?

Reply:
hmm yeah something is wonky, let me double check what happened here

Reply:
ok I figured it out, we are missing a backport of a doc change from master to 0.64. I'll make the change in the code repo. I've updated this PR with the fixed docs